switch.go

```diff
@@ -1,6 +1,6 @@
 package errorx
 
-// CaseNoTrait is a synthetic type used in TypeSwitch, signifying a presence of non-nil error of some other type.
+// NotRecognisedType is a synthetic type used in TypeSwitch, signifying casting error to *Error failed.
```

Comment:
Actually, not only that. Any type not expected in a switch, too.
So I suggest leaving the second half unchanged.